docs/reference/docs/reindex.asciidoc

```diff
@@ -576,6 +576,7 @@ The JSON response looks like this:
   "failures" : [ ]
 }
 --------------------------------------------------
+// NOTCONSOLE
```

Comment:
I think we can actually test this and the advantage of doing so is that we'll catch if the output of reindex changes.

Reply:
Right, this snippet is really out of date